themes/custom/droopler_theme/scss/components/_d_p_text_paged.scss

```diff
@@ -39,6 +39,7 @@ $d-p-text-paged-first-margin-top: 2% !default;
   .field--name-field-d-long-text {
     p {
       color: $d-p-text-paged-color;
+      word-wrap: break-word;
```

Comment:
Dorzuciłbym tu obsługę https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens - w Chrome nie zadziała, ale w pozostałych przeglądarkach zawsze będzie to jakieś ładniejsze wyjście awaryjne

Reply:
Ok, dodałem. hyphens.